Plot/Components/PartialComponents/Link/Link.razor

```diff
@@ -0,0 +1,17 @@
+<a id="@Id" href="@Url" class=" Link @Class">
+    @Text
+</a>
+
+@code {
+    [Parameter]
+    public string? Id { get; set; }
+
+    [Parameter]
+    public string? Class { get; set; }
+
+    [Parameter, EditorRequired]
+    public required string Text { get; set; }
```

Comment:
@ldwollen Could you change this to be using a child fragment so we can just write it like `<Link>Hello</Link>`

https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-9.0#child-content-render-fragments